toolchain/lowering/lowering_handle_expression_category.cpp

```diff
@@ -8,10 +8,27 @@ namespace Carbon {
 
 auto LoweringHandleBindValue(LoweringFunctionContext& context,
                              SemIR::NodeId node_id, SemIR::Node node) -> void {
-  // TODO: Take the value representation into account here.
-  context.SetLocal(node_id, context.builder().CreateLoad(
-                                context.GetType(node.type_id()),
-                                context.GetLocal(node.GetAsBindValue())));
+  auto source_id = node.GetAsBindValue();
```

Comment:
I'll note, this feels a little odd to have out here since it's not used on all the paths, and is short (could be inlined).

Reply:
Done.